File: tags/2010.8/examples/rest/team.rb
# To interact with teams on Planigle, use code like (Note: teams must have a story_id):
#   team = Team.create(:name => 'team a', :project_id => Project.find(:first, :name => "New Project").id)
#
#   teams = project.teams
#
#   team.name = 'team b'
#   team.save
#
#   team.destroy
#
# See remote.rb in this directory for more information on interacting with Planigle via REST.
#
# See attr_accessible in /app/models/team.rb for a list of allowed fields.  See story.rb in that same direcotry for valid status codes.

class Team < Resource
  self.prefix = Resource.site.to_s + "projects/:project_id/"
end

File: src/app/team.ts
export class Team {
  public id: number;
  public name: string;
  public description: string;
  public project_id: number;

  constructor(values: any) {
    this.id = values.id;
    this.name = values.name;
    this.description = values.description;
    this.project_id = values.project_id;
  }
}
